packages/kbn-management/settings/components/form/storybook/get_form_story.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { normalizeSettings } from '@kbn/management-settings-utilities';
+import { FieldDefinition, SettingType, UiSetting } from '@kbn/management-settings-types';
+import { getFieldDefinition } from '@kbn/management-settings-field-definition';
+import { Form } from '../form';
+import _settings from './settings.json';
+
+export const getFormStory = () => {
+  const Story = () => {
+    const settings = _settings as unknown as Record<string, UiSetting<SettingType>>;
+    const normalizedSettings = normalizeSettings(settings);
+    const fields: Array<FieldDefinition<SettingType>> = Object.entries(normalizedSettings).map(
+      ([id, setting]) =>
+        getFieldDefinition({
```

Comment:
nit: `getFieldDefinitions` from `@kbn/management-settings-field-definition` does this for you.

https://github.com/elastic/kibana/blob/main/packages/kbn-management/settings/field_definition/get_definitions.ts

Reply:
It seems that `getFieldDefinitions` requires passing a `IUiSettingsClient` to obtain the `isCustom` and `isOverridden` setting params. What would we pass if we were to use it here?